product_quantity_update_force_inventory/README.rst

```diff
@@ -0,0 +1,76 @@
+=======================================
+Product Quantity Update Force Inventory
+=======================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fstock--logistics--warehouse-lightgray.png?logo=github
+    :target: https://github.com/OCA/stock-logistics-warehouse/tree/13.0/product_quantity_update_force_inventory
+    :alt: OCA/stock-logistics-warehouse
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/stock-logistics-warehouse-13-0/stock-logistics-warehouse-13-0-product_quantity_update_force_inventory
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/153/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module block the ability to update Inventory quantities using Stock Quant Model.
```

Comment:
Small rewording of this section.

With this module, when a user goes to a product and presses the button 'Update Quantity' they will be directed to a new Inventory Adjustment for this product, instead of driving the user to update the quantities in the stock quants.

By doing this users will be able to have full traceability on the inventory adjustment activities.
 
Consider using this module together with the module 'stock_change_qty_reason' (https://github.com/OCA/stock-logistics-warehouse/tree/13.0/stock_change_qty_reason).

Reply:
comment attended @JordiBForgeFlow